lib/puppetserver/ca/action/prune.rb

```diff
@@ -49,55 +50,59 @@ def run(inputs)
           # Getting the CRL(s)
           loader = X509Loader.new(puppet.settings[:cacert], puppet.settings[:cakey], puppet.settings[:cacrl])
 
-          puppet_crl = loader.crls.select { |crl| crl.verify(loader.key) }
-          number_of_removed_duplicates = prune_CRLs(puppet_crl)
+          verified_crl = loader.crls.select { |crl| crl.verify(loader.key) }
```

Comment:
(nit) I would pluralize this, just so it's clear that this returns an array
```suggestion
          verified_crls = loader.crls.select { |crl| crl.verify(loader.key) }
```